corehq/apps/users/models.py

```diff
@@ -1073,6 +1073,14 @@ def wrap(cls, data, should_save=False):
             del data["user_data"]["commtrack-supply-point"]
             should_save = True
 
+        # Wrapping a doc from ES adds these extra fields which shouldn't be saved
+        if "__group_ids" in data:
```

Comment:
@sravfeyn FYI https://dimagi-dev.atlassian.net/browse/QA-1835

Reply:
```suggestion
        for attr in ["__group_ids", "__group_names", "user_data_es"]:
                if attr in data:
                        del data[attr]
```

Reply:
Hmm... I actually think saving docs that are coming in from ES is dangerous, since we could be saving an old version. I'm going to think about this a bit more.